README.md

```diff
@@ -140,7 +140,23 @@ Example:
 
 ```
 
+### variable replacement
+The `gp-translate` directive supports an optional `values` attribute that allows passing of custom data to it.  This is used to inject
+variables into the returned translation string.
 
+Example translation string:
+
+`"DATA-KEY-WITH-VARIABLE" : "Welcome {{user}}"`
+
+Example usage:
+
+```html
+
+    <gp-translate key="DATA-KEY-WITH-VARIABLE" bundle="someBundle" values="{{controllerAsVariable.userName}}"/>

```

Comment:
does this assume `controllerAsVariable.userName` is a map? Or does it just assume the only item will be substituted?